docs/index.js

```diff
@@ -0,0 +1,18 @@
+
+import SwaggerUI from 'swagger-ui';
+import SwaggerParser from "@apidevtools/swagger-parser";
+
+import 'swagger-ui/dist/swagger-ui.css';
+
+import spec from '../specs/openapi.json';
+
+const ui = SwaggerUI({
+    spec,
+    dom_id: '#swagger',
+});
+
+ui.initOAuth({
+    appName: "Swagger UI Webpack Demo",
```

Comment:
Not sure where this is used but should probably be

```suggestion
    appName: "Wikibase REST API Proposal",
```